crates/sp-domains-fraud-proof/src/host_functions.rs

```diff
@@ -97,6 +98,43 @@ where
         .ok()
         .map(|ext| ext.encode())
     }
+
+    fn is_tx_in_range(
+        &self,
+        consensus_block_hash_with_runtime_code: H256,
+        consensus_block_hash_with_tx_range: H256,
+        domain_id: DomainId,
+        opaque_extrinsic: OpaqueExtrinsic,
+        bundle_vrf_hash: U256,
+    ) -> Option<bool> {
+        let runtime_api = self.consensus_client.runtime_api();
+        let consensus_block_hash_with_tx_range = consensus_block_hash_with_tx_range.into();
+        let domain_tx_range = runtime_api
+            .domain_tx_range(consensus_block_hash_with_tx_range, domain_id)
+            .ok()?;
+
+        let consensus_block_hash_with_runtime_code = consensus_block_hash_with_runtime_code.into();
+        let runtime_code = runtime_api
+            .domain_runtime_code(consensus_block_hash_with_runtime_code, domain_id)
```

Comment:
there is already an existing function to get domain runtime code, can you use that pls ?